gradle/libs.versions.toml

```diff
@@ -10,7 +10,7 @@ systembartint = "1.0.3"
 robolectric = "4.9"
 glide = "4.14.2"
 sshj = "0.35.0"
-jcifs = "2.1.9"
+jcifs = "2.1.10"
```

Comment:
does this tag of jcifs solve the bouncycasle dependency issue? can you please share the reference?

Reply:
It's mentioned in the changelogs: https://github.com/AgNO3/jcifs-ng/blob/master/CHANGELOG.txt

Reply:
awesome, thanks! :)